src/DynamicData/Binding/IObservableListEx.cs

```diff
@@ -0,0 +1,197 @@
+            observableList = sourceList;
+
+            // Return a observable that will connect to the source so we can properly dispose when the pipeline ends.
+            return Observable.Create<IChangeSet<TObject>>(observer =>
+            {
+                return source
+                    .Finally(() => sourceList.Dispose())
+                    .SubscribeSafe(observer);
+            });
+        }
+
+        /// <summary>
+        /// Binds the results to the specified <see cref="IObservableList{T}"/>. Unlike
+        /// binding to a <see cref="ReadOnlyObservableCollection{T}"/> which loses the
+        /// ability to refresh items, binding to an <see cref="IObservableList{T}"/>.
+        /// allows for refresh changes to be preserved and keeps the list read-only.
+        /// </summary>
+        /// <typeparam name="TObject">The type of the object.</typeparam>
+        /// <typeparam name="TKey">The type of the key.</typeparam>
+        /// <param name="source">The source.</param>
+        /// <param name="destination">The destination.</param>
+        /// <returns>The <paramref name="source"/> changeset for continued chaining.</returns>
+        /// <exception cref="System.ArgumentNullException">source</exception>
+        public static IObservable<IChangeSet<TObject, TKey>> BindToObservableList<TObject, TKey>(
+            this IObservable<IChangeSet<TObject, TKey>> source,
+            out IObservableList<TObject> observableList)
+        {
+            if (source == null) throw new ArgumentNullException(nameof(source));
+
+            // Load our source list with the change set.
+            // Each changeset we need to convert to remove the key.
+            var sourceList = new SourceList<TObject>();
+
+            // Output our readonly observable list, preventing the sourcelist from being editted from anywhere else.
+            observableList = sourceList;
+
+            // Return a observable that will connect to the source so we can properly dispose when the pipeline ends.
+            return Observable.Create<IChangeSet<TObject, TKey>>(observer =>
+            {
+                return source
+                    .Do(changes => sourceList.Edit(editor => editor.Clone(changes.Convert(editor))))
+                    .Finally(() => sourceList.Dispose())
+                    .SubscribeSafe(observer);
+            });
+        }
+
+        /// <summary>
+        /// Binds the results to the specified <see cref="IObservableList{T}"/>. Unlike
+        /// binding to a <see cref="ReadOnlyObservableCollection{T}"/> which loses the
+        /// ability to refresh items, binding to an <see cref="IObservableList{T}"/>.
+        /// allows for refresh changes to be preserved and keeps the list read-only.
+        /// </summary>
+        /// <typeparam name="TObject">The type of the object.</typeparam>
+        /// <typeparam name="TKey">The type of the key.</typeparam>
+        /// <param name="source">The source.</param>
```

Comment:
Incorrect param name "destination" on the comments. Similar else where in this class

Reply:
Good catch, this has been fixed.